definitions.yaml

```diff
@@ -43,6 +43,9 @@ definitions:
       owner:
         type: string
         description: Name of the organization owning the cluster
+      release_version:
+        type: string
+        description: Release version to use after an upgrade
```

Comment:
Full stop is missing IMO.

Reply:
What means after an upgrade? It is the release version used to do the update to. We cannot add a release after the cluster is updated.

Reply:
The modifyCluster operation (the PATCH request to /v4/clusters/<cluster_id>/) that modifies the `release_version` attribute of the cluster is supposed to initiate a guest cluster upgrade. (We also plan automated updates. Unsure if these use the method described here. However, this is what I think users should use.)

So cluster current has e. g. `release_version=1.2.34`. User does

```
PATCH /v4/clusters/clstr/
{
  "release_version": "1.3.0"
}
```

which starts the cluster upgrade.

Reply:
I think you and me means the same. "after an upgrade" simply means: once the upgrade is done.